docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md

```diff
@@ -358,5 +358,5 @@ prime-api-client --insecure support-reviewed-payment-requests --filename payload
 
 Some helpful resources for getting this setup
 
-* [Acceptance Testing Payment Requests](Acceptance-Testing-Payment-Requests.md)
-* [Manually run Prime API for Slice demo](Manually-run-Prime-API-for-Slice-demo.md)
+* [Acceptance Testing Payment Requests](Acceptance-Testing-Payment-Requests)
+* [Manually run Prime API for Slice demo](Manually-run-Prime-API-for-Slice-demo)
```

Comment:
This page is located in a different directory `docs/dev/testing/running-tests/Manually-run-Prime-API-for-Slice-demo` / at this [url](https://transcom.github.io/mymove-docs/docs/dev/testing/running-tests/Manually-run-Prime-API-for-Slice-demo)

Reply:
I'll go check the urls again locally.